src/webgpu/web_platform/worker/worker.spec.ts

```diff
@@ -51,3 +51,21 @@ g.test('shared_worker')
     const result = await launchSharedWorker();
     assert(result.error === undefined, `should be no error from worker but was: ${result.error}`);
   });
+
+g.test('service_worker')
+  .desc(`test WebGPU is available in service workers and check for basic functionality`)
+  .fn(async t => {
+    if (isNode()) {
+      t.skip('node does not support 100% compatible workers');
+      return;
```

Comment:
Return is redundant because skip throws an exception. Also I think you can use:
`t.skipIf(isNode(), '...');`
instead.